web/blueprints/mpskclient/__init__.py

```diff
@@ -0,0 +1,158 @@
+        return default_response()
+
+    # existence verified by validator
+    # TODO can't we provide an attribute for that on the form?
+    owner = session.session.get(User, form.owner_id.data)
+    with abort_on_error(default_response), session.session.begin_nested():
+
+        host = lib_mpsk.mpsk_client_create(
+            owner, form.name.data, form.mac.data, processor=current_user
+        )
+    session.session.commit()
+
+    flash("MPSK Client erfolgreich erstellt.", "success")
+    return redirect(url_for("user.user_show", user_id=host.owner.id, _anchor="mpsks"))
+
+
+@bp.route("/<int:mpsk_id>/delete", methods=["GET", "POST"])
+@access.require("hosts_change")
+def mpsk_delete(mpsk_id: int) -> ResponseReturnValue:
+    mpsk = get_mpsk_client_or_404(mpsk_id)
+    owner = mpsk.owner
+    form = FlaskForm()
+
+    def default_response() -> ResponseReturnValue:
+        form_args = {
+            "form": form,
+            "cancel_to": url_for("user.user_show", user_id=owner.id),
+            "submit_text": "Löschen",
+            "actions_offset": 0,
+        }
+
+        return render_template(
+            "generic_form.html", page_title="MPSK Client löschen", form_args=form_args, form=form
+        )
+
+    if not form.is_submitted():
+        return default_response()
+
+    with abort_on_error(default_response), session.session.begin_nested():
+        lib_mpsk.mpsk_delete(mpsk, current_user)
+    session.session.commit()
+
+    flash("MPSK Client erfolgreich gelöscht.", "success")
+    return redirect(url_for("user.user_show", user_id=owner.id, _anchor="mpsks"))
+
+
+@bp.route("/<int:mpsk_id>/edit", methods=["GET", "POST"])
+@access.require("hosts_change")
+def mpsk_edit(mpsk_id: int) -> ResponseReturnValue:
+    mpsk = get_mpsk_client_or_404(mpsk_id)
+
+    form = WiFiInterfaceForm(obj=mpsk)
+
+    def default_response() -> ResponseReturnValue:
+        form_args = {"form": form, "cancel_to": url_for("user.user_show", user_id=mpsk.owner_id)}
+
+        return render_template(
+            "generic_form.html", page_title="MPSK Client editieren", form_args=form_args
+        )
+
+    if not form.is_submitted() or not form.validate():
+        return default_response()
+
+    with abort_on_error(default_response), session.session.begin_nested():
+        lib_mpsk.mpsk_edit(mpsk, mpsk.owner, form.name.data, form.mac.data, current_user)
+    session.session.commit()
+    flash("MPSK Client erfolgreich bearbeitet.", "success")
+    return redirect(url_for("user.user_show", user_id=mpsk.owner_id, _anchor="mpsks"))
+
+
+@bp.route("/<int:user_id>")
+def user_clients_json(user_id: int) -> ResponseReturnValue:
+    user = get_user_or_404(user_id)
+    # TODO: Importend when the for the returning of actual mpsk mac addresses for MPSK devices
+    # return ""
+    return TableResponse[MPSKRow](
+        items=[_mpsk_row(mpsk, user_id) for mpsk in user.mpsks]
+    ).model_dump()
+
+
+def _mpsk_row(client, user_id: int) -> MPSKRow:
```

Comment:
```suggestion
def _mpsk_row(client: MPSKClient, user_id: int) -> MPSKRow:
```
(and then add an import for `MPSKClient`)